pkgs/applications/misc/bambu-studio/default.nix

```diff
@@ -166,6 +166,12 @@ stdenv.mkDerivation rec {
     )
   '';
 
+  # needed to prevent collisions between the LICENSE.txt files of
+  # bambu-studio and orca-sliver.
+  postInstall = ''
+    mv $out/LICENSE.txt $out/LICENSE-${finalAttrs.pname}.txt
```

Comment:
```suggestion
     mkdir -p $out/share/doc 
     mv $out/LICENSE.txt $out/share/doc/LICENSE-${finalAttrs.pname}.txt
```

Reply:
can we drop `finalAttrs.pname` ? now that there is no clash, we can just keep name as intended by upstream

Reply:
True. I fixed it in an edit